### ISSUE TITLE
Crash in PPBluetoothAdvDeviceModel when calling [PPBluetoothAdvDeviceModel description]

### ISSUE THREAD

**Issue by churowa** [LefuHengqi/PPBaseKit#1]
I'm trying to use the new version that allows for simulator architechture support. First, thank you for delivering the simulator architecture, it makes me very happy.

I imported both PPBaseKit and PPBluetoothKit into my project, and it works, but I'm seeing the following crash when finding my device:

```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[PPBluetoothAdvDeviceModel PPAston_JSONString]: unrecognized selector sent to instance 0x302a58640'
*** First throw call stack:
(0x186b7108c 0x183e732e4 0x186c777c8 0x186b0e888 0x186b0e1b0 0x102bf5f84 0x102bf64f8 0x102c1f344 0x1aee21068 0x1aee1fb18 0x1aee1f9fc 0x1aee1f8a8 0x104108a30 0x10410a71c 0x1041125e8 0x104113394 0x10411acbc 0x10411a9a4 0x186b43f64 0x186b41188 0x186b405b8 0x1d25d61c4 0x1896965f0 0x18974510c 0x189a71e50 0x102be76b4 0x102be762c 0x102be7730 0x1ac313d34)
libc++abi: terminating due to uncaught exception of type NSException
```

PPBaseKit: 17fcbce2492160ff749288c0210b25963f0aeee3
PPBluetoothKit: 72cab228c73a0607a762702c21ef3146542874a8

Device: CK780BLE

### Steps to reproduce:
1. `[PPBluetoothConnectManager -surroundDeviceDelegate:self]`
2. `[PPBluetoothConnectManager -searchSurroundDevice]`
3. `[MyClass centralManagerDidFoundSurroundDevice:PPBLuetoothAdvDeviceModel:CBPeripheral]`
4. call `[PPBluetoothAdvDeviceModel -description]`
5. Observe crash; PPBluetoothAdvDeviceModel was not nil

### Example code:
```Swift
extension PPScanStreamHandler: PPBluetoothUpdateStateDelegate, PPBluetoothSurroundDeviceDelegate{
    func centralManagerDidUpdate(_ state: PPBluetoothState) {
        print("SS STATE \(state)")
        if (state == .poweredOn){
            // We know that bluetooth is on, so tell the SDK to look for devices
            if (!scanInProgreess) {
                print("SS SEARCH SURROUND DEVICE")
                ppConnectionManager?.searchSurroundDevice()
                scanInProgreess = true
            }
        }
    }
    
    func centralManagerDidFoundSurroundDevice(_ device: PPBluetoothAdvDeviceModel!, andPeripheral peripheral: CBPeripheral!) {
        os_log("PPScanStreamHandler:centralManagerDidFoundSurroundDevice", log:scanLog, type: .debug)
        os_log("SS Found Device: %@", log: scanLog, type: .debug, device.description) // <--- CRASH HERE
        let wrappedDevice = PPDeviceWrapper(ppDeviceModel: device, peripheral: peripheral)
        // We found a device, tell our SdkManager about it
        os_log("PPScanStreamHandler:centralManagerDidFoundSurroundDevices:onScalesFound - %@", log: scanLog, type: .debug, (onScalesFound != nil) ? "exists" : "nil")
        onScalesFound?([wrappedDevice])
        scanInProgreess = false
    }
}
```

**Comment by LefuHengqi** [comment]
@churowa 
I tried multiple devices. After successfully discovering the device, I called the "device.description" method and there was no crash. Can you provide more detailed information? For example: 1. What iPhone model and operating system are you running? 2. What is the "Bluetooth name" of the device in your hand?
SDK version I tested:
pod 'PPBluetoothKit', '1.2.12'
pod 'PPBaseKit', '1.2.4'

**Comment by churowa** [comment]
Ok... I think this is my fault. I saw your commits and got too excited. Initially, I just included the XCFrameworks from PPBluetoothKit and PPBaseKit directly into my project without using Cococapods.

I just re-tested after installing with Cocoapods as your instructions stated, and I'm not seeing the crash anymore. Everything seems perfect.

For what it's worth,

I tested on the following devices:
iPhone XR (iOS 18.0)
iPhone 13 Pro (iOS 18.0)
iPad (9th generation) (iOS 17.2)

{"PeripheralType":7,"deviceMac":"CA:E7:F3:15:00:02","deviceAccuracyType":3,"deviceConnectType":1,"deviceSettingId":25,"devicePower":-1,"deviceCalcuteType":6,"deviceFuncType":1,"deviceType":4,"deviceName":"LFSc","devicePowerType":1,"deviceUnitList":"4,7,8,1","rssi":-51,"macAddressStart":7,"deviceProtocolType":2,"advLength":20,"standardType":0,"sign":"CA0108"}

I'm going to close this, since using cocoapods solves the problem.

Thank you for your response, I think we're all good now.
Simulator works ✅
No crashing ✅
